data.sql

```diff
@@ -54,3 +54,46 @@ SET owner_id = (
   END
 );
 
+
+
+-- Vet clinic database: add "join table" for visits
+
+INSERT INTO vets (name, age, date_of_graduation)
+VALUES
+  ('William Tatcher', 45, '2000-04-23'),
+  ('Maisy Smith', 26, '2019-01-17'),
+  ('Stephanie Mendez', 64, '1981-05-04'),
+  ('Jack Harkness', 38, '2008-06-08');
+
+-- Insert specializations data
+INSERT INTO specializations (vet_id, species_id)
+VALUES
+  ((SELECT id FROM vets WHERE name = 'William Tatcher'), (SELECT id FROM species WHERE name = 'Pokemon')),
+  ((SELECT id FROM vets WHERE name = 'Stephanie Mendez'), (SELECT id FROM species WHERE name = 'Digimon')),
+  ((SELECT id FROM vets WHERE name = 'Stephanie Mendez'), (SELECT id FROM species WHERE name = 'Pokemon')),
+  ((SELECT id FROM vets WHERE name = 'Jack Harkness'), (SELECT id FROM species WHERE name = 'Digimon'));
+
+-- Insert visits data with vet_id and animal_id
+INSERT INTO visits (vet_id, animal_id, visit_date)
```

Comment:
### Optional
- [ ] Once we create the `visit_date` column in the `Visits` table, we can successfully insert these data without errors